README.md

```diff
@@ -1 +1,100 @@
-# Team-Kanban
\ No newline at end of file
+## TODO LIST WEB_APP
+
+- This is a JavaScript meal app built with crowd-sourced database of Recipes from around the world.
+
+## LIVE DEMO
+
+- Click [HERE]() 
+
+## Built With
+
+- HTML
+
+- CSS
+
+- Javascript(ES6)
+
+- WebPack
+
+- Fetch API
+
+- MealDB API
+
+- Involement API
+
+- Lighthouse (An open-source, automated tool for improving the quality of web pages. It has audits for performance, accessibility, progressive web apps, SEO and more).
+
+- Webhint (A customizable linting tool that helps you improve your site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors).
+
+- Stylelint (A mighty, modern linter that helps you avoid errors and enforce conventions in your styles).
+
+### Linter Tests
+
+- `npx hint .`
+- `npx stylelint "**/*.{css,scss}"`
+- `npx eslint .`
+
+### Prerequisites
+
+- Have a local version control like Git. Which is a open source distributed version control system designed for source code management
+- A text editor (e.g Visual Studio Code, Vim, Atom & Sublime.)
+- A web browser (e.g Chrome, Safari, Mozilla Firefox.)
+- Live Server which is a web extension that helps you to live reload feature for dynamic content (PHP, Node.js, ASPNET)
+- Download waves to check project accessibilty
+
+### Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- npm
+  ```sh
+  npm install npm@latest -g
+  ```
+
+### Installation
+
+1. Clone the repo by running the command
+   ```sh
+    git clone https://github.com/V-Blaze/FOODHUB.git
+   ```
+2. Open the directory of the project
+   ```sh
+   cd FOODHUB
+   ```
+3. Open the html file
+   ```sh
+   Click and open the html file
+   ```
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+See the [open issues](#) for a full list of proposed features (and known issues).
+
+## Authors
+
+👤 Valetine Blaze
+
+- GitHub: [@V-blaze](https://github.com/V-Blaze)
+- Twitter: [@blaze_valentine](https://twitter.com/blaze_valentine)
+- LinkedIn: [@valentine-blaze](https://www.linkedin.com/in/valentine-blaze/)
+
+## 
+
+👤 Usama Malik
+
+- GitHub: [@usamamaliq](https://github.com/usamamaliq)
+- LinkedIn: [@usamamaliq](https://www.linkedin.com/in/usamamaliq/)
+
+## 🤝 Contributing
+
+- Contributions, issues, and feature requests are welcome!
+
+- Feel free to check the [issues page](https://github.com/V-Blaze/FOODHUB.git/issues).
+
+## Show your support
+
+- Give a ⭐️ if you like this project!
+
+## 📝 License
+
+- This project is [MIT](https://github.com/V-Blaze/FOODHUB.git/blob/main/LICENSE) licensed.
```

Comment:
![image](https://user-images.githubusercontent.com/101483956/203804806-b16f1032-3d8c-4909-b7e4-294ab861204e.png)
The license file is not linked properly. Clicking the link gives an error. You can fix this by renaming the "LICENSE" file to "LICENSE.md" and updating the link in the readme as well.